### ISSUE TITLE
Option to disable contractions

### ISSUE THREAD

**Issue by AurielleP** [spencermountain/compromise#735]
Currently a pattern such as `12345-1234` is getting marked as `Contraction` and implicitly expanded. 

Ideally this should not happen - or there should be a method to undo expansion on contractions like those

**Comment by spencermountain** [comment]
thanks. The regex is [right here](https://github.com/spencermountain/compromise/blob/master/src/02-tagger/03-contractions/06-ranges.js#L1) if you, or anyone, wanted to do a pr.
cheers

**Comment by AurielleP** [comment]
I was thinking it would be helpful, instead of fixing that edge case - to have the ability to leave implicit as is - and prevent automatic expansion on ranges, if possible - maybe as an option that is passed in

one other idea might be, if the start of the range is not a lower number than the end of the range - its not valid..

**Comment by spencermountain** [comment]
changed numberRange to require < 4 numbers

new behaviour in next release:
```
    ['1-1', ['NumberRange', 'NumberRange', 'NumberRange']],
    ['12-12', ['NumberRange', 'NumberRange', 'NumberRange']],
    ['123-123', ['NumberRange', 'NumberRange', 'NumberRange']],
    ['1234-1234', ['Noun']],
```

**Comment by spencermountain** [comment]
this is now fixed in `13.6.0`
